### ISSUE TITLE
Does not detect a syntax error in this

### ISSUE THREAD

**Issue by Torniojaws** [AtomLinter/linter-js-yaml#135]
```
{%- set functions =
  {
    SomeFunction: {
      SomeParam: "Hello",
      Example": "this row has a syntax error!"
    }
  }
%}
```

The row with the variable **Example** has an extra quote before the colon (``Example":`` instead of correct ``Example:``) which will crash the yaml, but is not detected by linter-js-yaml.

**Comment by Torniojaws** [comment]
But I'm not sure if the yaml linter should catch these extra functions within a regular yaml file.

**Comment by cbho** [comment]
The file name is .yml when looking at the picture - isn't that the problem?

**Comment by Arcanemagus** [comment]
> The file name is .yml when looking at the picture - isn't that the problem?

I'm not sure what you are asking here?

----

Marking this as closed due to a lack of response from @Torniojaws. Thanks for taking the time to file an issue and if you are still experiencing issues feel free to comment further!
